forms-flow-web/src/components/Form/Item/View.js

```diff
@@ -57,7 +57,7 @@ import { setFormStatusLoading } from "../../../actions/processActions";
 import isEqual from "lodash/isEqual";
 
 const View = React.memo((props) => {
-  const [formStatus, setFormStatus] = useState("");
+  const [formStatus, setFormStatus] = React.useState("");
```

Comment:
useState is already imported right ? Can use it directly rather thaan React.

Reply:
That's for the test. `useState`  can be mocked only this way